holidayextras/ui-toolkit
# ATOMIC - Update the docs to split stuff into atomic design principles

#### What does this PR do? (please provide any background)
- We will be following Brad Frosts [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/) principles, which means we need to clearly distinguish our atoms and molecules in the UI-Toolkit.
- This has meant changing the NAV on the right and re-organising our components
- I have removed Justified container from the Toolkit as this doesn't fit with the atomic design principles

#### What tests does this PR have?
None (just updated names)

#### How can this be tested?
run `grunt docs` and and take a look at the way the components are laid out in the docs provided, Check that everything under the ATOMS header fits in with its [description](http://bradfrost.com/blog/post/atomic-web-design/#atoms) and that everything under the MOLECULES header fits in with it [description](http://bradfrost.com/blog/post/atomic-web-design/#molecules)

#### Screenshots / Screencast
<img width="129" alt="screen shot 2015-09-14 at 10 01 07" src="https://cloud.githubusercontent.com/assets/778942/9846058/8a7cd920-5ac7-11e5-8aeb-9ada38ae79d5.png">

#### What gif best describes how you feel about this work?
![Atomic](https://media.giphy.com/media/kTKhjmiUF0lri/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
